### DISCUSSION TITLE
[Request] Sway Support

### DISCUSSION THREAD

**Discussion by margaretjoanmiller** [Fabric-Development/fabric#92] [General]
Would it be possible to add sway support/is there an easy way to hack it in? It looks like the workspace class is pretty directly tied to hyprland in the current manifestation?

**Comment by its-darsh** [comment]
next couple of commits will introduce new features to Fabric, I've noted your request and will try to bring it when i find the time, for now you can use a `Fabricator` around the `swaymsg` command or use [this library](https://github.com/altdesktop/i3ipc-python)

**Comment by margaretjoanmiller** [comment]
Oh rad! thanks!
